spec/active_force/sobject_spec.rb

```diff
@@ -65,13 +65,33 @@
       end
     end
 
-    describe 'having an default value' do
-      it 'uses the default value' do
-        expect(Bangwhiz.new[:percent]).to eq(50.0)
+    describe 'having a default value' do
+      subject { Bangwhiz }
+
+      shared_context 'new object with a default field' do
```

Comment:
This is just a personal preference, so feel free to ignore.  I think these tests are unnecessarily DRY.  I think it would be easier to read and understand them if they were duplicated in the two contexts instead of storing them away in this `shared_context`.  And the reliance on that externally provided `let(:percent)` and `let(:instantiation_attributes)` variables makes the shared context itself a little difficult to understand--I couldn't tell immediately where they were coming from when looking at the shared_context.

Reply:
Thanks for the feedback here, wanted to try something new with the shared_context. Good to know it reduced the overall communication.

In other scenarios, do you have any advice for combatting deeply nested context blocks?